pages/store/[name]/source.js

```diff
@@ -0,0 +1,134 @@
+import { useState, useEffect } from 'react'
+import Head from 'next/head'
+import { useRouter } from 'next/router'
+import plugins from 'plugins.json'
+import Page from 'components/page'
+import PluginInfo from 'components/plugin-info'
+import { File, Directory } from 'components/icons'
+import useSWR from 'swr'
+import styles from 'styles/pages/store/source.module.css'
+
+const formatFileName = (path) => path.replace(/^\/+|\/+$/g, '')
+
+export default () => {
```

Comment:
Can you add SSG to this page too 🙏 

The source code won't change that much for plugins, and you can add `unstable_revalidate` to SSG, the page will be revalidated in the background after X seconds